ts-generator.json

```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "generator":"typechain",
+    "files":"./build/**/*/!(Migrations.json|MessageBus.json|BytesLib.json|SafeMath.json|MetaBlock.json|RLP.json|RLPEncode.sol|*Mock.json|Mock*|Test*|*Test.json|*Interface.json|Spy*)",
```

Comment:
Why `MessageBus.json` is excluded?
I think any external library should be included.

Reply:
It's a known issue in TypeChain.
https://github.com/ethereum-ts/TypeChain/issues/125

It seems there is no support for complex types e.g. structs(MessageBus.MessageBox) for library contracts.